frontend/src/components/subject/subjectview/header/SubjectHeaderCard.vue

```diff
@@ -1,28 +1,41 @@
 <template>
     <v-card variant="text" class="title-card" height="165">
         <v-row>
-            <v-col :cols="isAdmin || (isInstructor && isTeacher) ? 10 : 12">
+            <v-col :cols="isAdmin || (isInstructor && isTeacher) ? 11 : 12" class="scrollable-col">
                 <v-card-title class="title">
-                    {{ title }}
+                    <div class="scrollable">
```

Comment:
nice dat dit scrollable is, maar niet echt duidelijk voor de gebruiker dat dit mogelijk is

Reply:
wordt gefixt door @masinnae normaal